src/client/main.c

```diff
@@ -34,24 +34,26 @@ int method_version(UNUSED Command *command, UNUSED void *userdata) {
-        { "thaw",            2, 2,       OPT_NONE,                                method_thaw,            usage_bluechi },
-        { "restart",         2, 2,       OPT_NONE,                                method_restart,         usage_bluechi },
-        { "reload",          2, 2,       OPT_NONE,                                method_reload,          usage_bluechi },
-        { "monitor",         0, 2,       OPT_NONE,                                method_monitor,         usage_bluechi },
-        { "metrics",         1, 1,       OPT_NONE,                                method_metrics,         usage_bluechi },
-        { "enable",          2, ARG_ANY, OPT_FORCE | OPT_RUNTIME | OPT_NO_RELOAD, method_enable,          usage_bluechi },
-        { "disable",         2, ARG_ANY, OPT_NO_RELOAD,                           method_disable,         usage_bluechi },
-        { "daemon-reload",   1, 1,       OPT_NONE,                                method_daemon_reload,   usage_bluechi },
-        { "status",          0, ARG_ANY, OPT_WATCH,                               method_status,          usage_bluechi },
-        { "set-loglevel",    1, 2,       OPT_NONE,                                method_set_loglevel,    usage_bluechi },
-        { "version",         0, 0,       OPT_NONE,                                method_version,         usage_bluechi },
-        { NULL,              0, 0,       0,                                       NULL,                   NULL          }
+        { "help",              0, 0,       OPT_NONE,                                method_help,              usage_bluechi },
+        { "list-unit-files",   0, 1,       OPT_FILTER,                              method_list_unit_files,   usage_bluechi },
+        { "list-units",        0, 1,       OPT_FILTER,                              method_list_units,        usage_bluechi },
+        { "start",             2, 2,       OPT_NONE,                                method_start,             usage_bluechi },
+        { "stop",              2, 2,       OPT_NONE,                                method_stop,              usage_bluechi },
+        { "freeze",            2, 2,       OPT_NONE,                                method_freeze,            usage_bluechi },
+        { "thaw",              2, 2,       OPT_NONE,                                method_thaw,              usage_bluechi },
+        { "restart",           2, 2,       OPT_NONE,                                method_restart,           usage_bluechi },
+        { "reload",            2, 2,       OPT_NONE,                                method_reload,            usage_bluechi },
+        { "reset-failed",      1, 1,       OPT_NONE,                                method_reset_failed,      usage_bluechi },
+        { "reset-failed-unit", 2, 2,       OPT_NONE,                                method_reset_failed_unit, usage_bluechi },
```

Comment:
I agree with @mkemel 
@ArtiomDivak Please remove the `reset-failed-unit` method. Everything should be handled by `reset-failed`. Lets define the desired behavior:
```bash
# resetting failed state of all units on node x
bluechictl reset-failed <node x>

# resetting failed state of unit1, unit2, etc. on node x
bluechictl reset-failed <node x> <unit1> <unit2> ...

# resetting failed state of all units on all nodes
bluechictl reset-failed
```
What do you think? Are these three reset-failed features in bluechictl ok? @mkemel @ArtiomDivak 
I'd skip implementing resolving globs for now - if needed, it can be done later on as well.

Reply:
SGTM

Reply:
We can open a low-prio issue for glob resolving. This can be implemented similarly to the it is done in `systemctl` and used for other methods as well where applies

Reply:
Created a follow up issue: https://github.com/eclipse-bluechi/bluechi/issues/936
@ArtiomDivak @mkemel PTAL

Resolving this conversation.